tests/src/system/basic/CLIActionTests.java

```diff
@@ -1,501 +0,0 @@
-          assertTrue("Expected result to not contain error", !result.contains("error"));
-
-          wsk.delete(Action, action);
-          wsk.createAction(action, TestUtils.getCatalogFilename("samples/hello.js"));
-
-          activationId = wsk.invoke(action, TestUtils.makeParameter("payload", now));
-          present = wsk.logsForActivationContain(activationId, expectedFirstResult, DEFAULT_WAIT);
-          assertTrue("Unexpected '" + expectedFirstResult + "' which is in log for activation " + activationId, !present);
-          present = wsk.logsForActivationContain(activationId, expectedSecondResult, DEFAULT_WAIT);
-          assertTrue("Expected '" + expectedSecondResult + "' which is missing in log for activation " + activationId, present);
-      } finally {
-          wsk.delete(Action, action);
-      }
-    }
-
-    @Ignore
-    @Test(timeout=120*1000)
-    public void invokeActionWithSpecialCharacters() throws Exception {
-        String action = "invokeActionWithUTF8";
-        try {
-            wsk.sanitize(Action, action);
-            wsk.createAction(action, TestUtils.getCatalogFilename("samples/hello.js"));
-            String msg = "«ταБЬℓσö»: 1<2 & 4+1>³, now 20%€§$ off!";
-            String expected = "hello " + msg;
-            String activationId = wsk.invoke(action, TestUtils.makeParameter("payload", msg));
-            boolean present = wsk.logsForActivationContain(activationId, expected, DEFAULT_WAIT);
-            assertTrue("Expected '" + expected + "' which is missing in log for activation " + activationId, present);
-        } finally {
-            wsk.delete(Action, action);
-        }
-    }
-
-    @Test(timeout = 120 * 1000)
-    public void testPingNotAllowed() throws Exception {
-        String action = "testPingNotAllowed";
-        try {
-            wsk.sanitize(Action, action);
-            wsk.createAction(action, TestUtils.getTestActionFilename("ping.js"));
-            String hostToPing = "google.com";
-            long startInvoke = System.currentTimeMillis();
-            String activationId = wsk.invoke(action, TestUtils.makeParameter("payload", hostToPing));
-            String expected = "ping: icmp open socket: Operation not permitted";
-            wsk.activationsContain(action, activationId, startInvoke, DEFAULT_WAIT);
-            String result = wsk.getResult(activationId).stdout.trim();
-            assertTrue("Expected result of " + activationId + " to be '" + expected + "' non-empty but is '" + result + "'", result.contains(expected));
-        } finally {
-            wsk.delete(Action, action);
-        }
-    }
-
-    /**
-     * Test creating an action from a js file always returning an application error.
-     * The invocation should succeed, the activation result should contain an error as
-     * defined in the action and the logs should be empty.
-     */
-    @Test
-    public void applicationError() throws Exception {
```

Comment:
Basically same as "it should "create and invoke a blocking action resulting in a whisk.error response"" in WskBasicTests.scala?